api/CSS.json

```diff
@@ -63,7 +63,7 @@
             "version_added": null
           },
           "webview_android": {
-            "version_added": "66"
+            "version_added": "37"
```

Comment:
Since `webview_android` became Chrome-based, most all of its feature support should be the same as `chrome_android`.  The `supports` subfeature indicates `chrome_android` support, and the `webview_android` block is a copy-paste of that with an adjustment of bumping "28" up to "37".

Of course, I could be wrong in this assumption.  :P